data/tabular/qm9/meta.yaml

```
@@ -0,0 +1,187 @@
+    - id: g298
+      description: free energy at 298.15 K
+      units: Hartree
+      type: float
+      names:
+          - noun: free energy at 298.15 K
+          - noun: gibbs free energy at 298.15 K
+    - id: heat_capacity
+      description: heat capacity at 298.15 K
+      units: cal/(mol K)
+      type: float
+      names:
+          - noun: heat capacity at 298.15 K
+identifiers:
+    - id: smiles
+      type: SMILES
+      description: SMILES
+    - id: inchi
+      type: InChI
+      description: International Chemical Identifier
+license: CC0 1.0
+num_points: 133885
+links:
+    - url: https://figshare.com/collections/Quantum_chemistry_structures_and_properties_of_134_kilo_molecules/978904
+      description: original data source
+    - url: https://huggingface.co/datasets/n0w0f/qm9-csv/blob/main/qm9_dataset.csv
+      description: parsed dataset in csv format
+bibtex:
+    - |-
+      @article{ramakrishnan2014quantum,
+      title={Quantum chemistry structures and properties of 134 kilo molecules},
+      author={Ramakrishnan, Raghunathan and Dral, Pavlo O and Rupp, Matthias and Von Lilienfeld, O Anatole},
+      journal={Scientific data},
+      volume={1},
+      number={1},
+      pages={1--7},
+      year={2014},
+      publisher={Nature Publishing Group}}
+    - |-
+      @article{ruddigkeit2012enumeration,
+      title={Enumeration of 166 billion organic small molecules in the chemical universe database GDB-17},
+      author={Ruddigkeit, Lars and Van Deursen, Ruud and Blum, Lorenz C and Reymond, Jean-Louis},
+      journal={Journal of chemical information and modeling},
+      volume={52},
+      number={11},
+      pages={2864--2875},
+      year={2012},
+      publisher={ACS Publications}}
+templates:
+    - The molecule with the {smiles__description} representation of {smiles#} has a dipole moment of {dipole_moment#} Debye.
+    - The molecule with the {inchi__description} representation of {inchi#} has a dipole moment of {dipole_moment#} Debye.
+    - The molecule with the {smiles__description} representation of {smiles#} has an isotropic polarizability of {polarizability#} Bohr^3.
+    - The molecule with the {inchi__description} representation of {inchi#} has an isotropic polarizability of {polarizability#} Bohr^3.
+    - The molecule with the {smiles__description} representation of {smiles#} has an energy of highest occupied molecular orbital {homo#} Hartree.
+    - The molecule with the {inchi__description} representation of {inchi#} has an energy of highest occupied molecular orbital {homo#} Hartree.
+    - The molecule with the {smiles__description} representation of {smiles#} has an energy of lowest occupied molecular orbital {lumo#} Hartree.
+    - The molecule with the {inchi__description} representation of {inchi#} has an energy of lowest occupied molecular orbital {lumo#} Hartree.
+    - The molecule with the {smiles__description} representation of {smiles#} has a homo-lumo gap of {gap#} Hartree.
```

```suggestion
    - The molecule with the {smiles__description} representation of {smiles#} has a HOMO-LUMO gap of {gap#} Hartree.
```

i guess this is the conventional capitalization?